server/platform/services/sharedchannel/sync_send_remote.go

```diff
@@ -35,6 +35,7 @@ type syncData struct {
 	profileImages map[string]*model.User
 	posts         []*model.Post
 	reactions     []*model.Reaction
+	statuses      []*model.Status
```

Comment:
Currently we're only sending one status per message, but this is a list in case we decide to optimize further down the line and send multiple, so the API doesn't change and we keep compatibility. The server always processes the whole list when a message is received, so old servers should process messages with multiple statuses correctly